safe/definitions.py

```diff
@@ -356,12 +356,22 @@
         'bridge.')
 }
 
+exposure_area = {
+    'key': 'area',
+    'name': tr('Area'),
+    'description': tr(
+        'The <b>area</b> exposure data describes features on '
+        'the surface of the earth that might be exposed to a particular '
+        ' hazard. This currently include urban areas. ')
```

Comment:
My personal taste :) : please avoid putting space in the beginning of a string. It will make the code prettier. In this case, there will be double space between `particular` and `hazard`